drivers/timer/cortex_m_systick.c

```diff
@@ -87,14 +87,16 @@ void z_clock_isr(void *arg)
 {
 	ARG_UNUSED(arg);
 	u32_t dticks;
+	u32_t bool_cnt_flag;
 
-	cycle_count += last_load;
+	bool_cnt_flag = (SysTick->CTRL & SysTick_CTRL_COUNTFLAG_Msk); /* Reset overflow flag */
+	bool_cnt_flag >>= SysTick_CTRL_COUNTFLAG_Pos;
+	overflow_cyc = 0;
+
+	cycle_count += last_load * bool_cnt_flag;
```

Comment:
Yes I've wrote it this way to avoid a jump condition, but it has no advantage in thumb code.

Will change it to a?b:c as in other places in this file.